changelog.txt

```diff
@@ -1,3 +1,10 @@
+Version 3.8.0, November 2, 2020
+
+Renamed CFAbstractTransfer.getValueWithSameAnnotations() to two methods
```

Comment:
I wouldn't call this a renaming.  `CFAbstractTransfer.getValueWithSameAnnotations()` was removed.  Also, we don't necessarily need to remove that method.  If we keep it in, then this change wouldn't be a breaking change.

Reply:
@smillst I think it's still a breaking change because `getValueWithSameAnnotations()` is no longer called.  If we leave it in, then clients of the Checker Framework will continue to compile, but their behavior will change.  If you think that's OK, we could make the version number 3.7.1.  What do you think?

Reply:
The documentation of `getValueWithSameAnnotations` doesn't say that it is called for narrowing and widening conversions, so clients shouldn't rely on that fact. So, I'm ok with this.